src/utils.js

```diff
@@ -4,3 +4,11 @@
 // FIXME: Could not find a way to define a Callback<T> type with multiple
 //        signatures.
 export type Callback = (?Error, any) => void;
+
+export function composeAsync (f1: (...args: Array<*>) => Promise<*>,
+                       f2: (*) => Promise<*>):
+                      (...args: Array<*>) => Promise<*> {
+  return async function composed (...args: Array<*>): Promise<*> {
+    return f2(await f1(...args))
```

Comment:
Well, actually, it doesn't reject, but it does throw...
So even if the test was broken, doesn't the code still works?
I mean, some unexpected exception is thrown, be it a bug or a custom exception type.
And it bubbles up to the unit test, instead of having the function rejecting the promise...
Is it really an issue?
(I may be missing something)